gpjax/sparse_gps.py

```diff
@@ -0,0 +1,169 @@
+from typing import Callable, Dict, Optional
+
+import distrax as dx
+import jax.numpy as jnp
+from chex import dataclass
+from jax import vmap
+from jax.numpy.linalg import cholesky
+from jax.scipy.linalg import solve_triangular
+
+from gpjax.config import get_defaults
+from gpjax.gps import Posterior
+from gpjax.kernels import cross_covariance, gram
+from gpjax.parameters import transform
+from gpjax.types import Array, Dataset
+from gpjax.utils import I, concat_dictionaries
+
+from .quadrature import gauss_hermite_quadrature
+from .variational import VariationalFamily, VariationalGaussian, VariationalPosterior
+
+DEFAULT_JITTER = get_defaults()["jitter"]
+
+
+def SVGP(posterior: Posterior, variational_family: VariationalFamily) -> VariationalPosterior:
```

Comment:
I'm uncomfortable with functions being capitalised - it makes it look like a class. Can we name this something else?

Reply:
Addressed in [be2eb5d](https://github.com/thomaspinder/GPJax/pull/43/commits/be2eb5da0b179efba6ddeee4c74ebe4232e56080).